src/Config.js

```diff
@@ -100,6 +106,17 @@ module.exports = class extends ChainedMap {
               .forEach(name => this.plugin(name).merge(value[name]));
           }
 
+          case 'plugins': {
+            if (Array.isArray(value)) {
+              return value
+                .forEach((plugin, index) => {
+                  if (plugin.name) index = plugin.name;
+                  if (isClass(plugin)) this.plugin(index).plugin(plugin);
```

Comment:
Instead of pulling in another dependency, couldn't this just check if the plugin is a function?

```js
if (typeof plugin === 'function') {
  this.plugin(index).plugin(plugin);
}
```